plugins/@grouparoo/iterable/__tests__/export/destination-mapping-options.ts

```diff
@@ -0,0 +1,75 @@
+import * as path from "path";
+import "@grouparoo/spec-helper";
+import { helper } from "@grouparoo/spec-helper";
+
+import {
+  destinationMappingOptions,
+  getUserFields,
+} from "../../src/lib/export/destinationMappingOptions";
+import { loadAppOptions, updater } from "../utils/nockHelper";
+import { connect } from "../../src/lib/connect";
+
+const nockFile = path.join(
+  __dirname,
+  "../",
+  "fixtures",
+  "destination-mapping-options.js"
+);
+
+// these comments to use nock
+// const newNock = false;
+// require("./../fixtures/destination-mapping-options");
+// or these to make it true
+const newNock = true;
+helper.recordNock(nockFile, updater);
+
+const appOptions = loadAppOptions(newNock);
+
+async function runDestinationMappingOptions({}) {
+  return destinationMappingOptions({
+    appOptions,
+    app: null,
+    appGuid: null,
+    connection: null,
+    destination: null,
+    destinationGuid: null,
+    destinationOptions: null,
+  });
+}
+
+describe("iterable/destinationMappingOptions", () => {
+  test("can fetch user fields", async () => {
+    const client = await connect(appOptions);
+    const fields = await getUserFields(client);
+    const text = fields.find((f) => f.key === "phoneNumber");
```

Comment:
these variable names don't look quite right. I would think this would would be `phoneNumber` too.